### ISSUE TITLE
NodeJS Lessons: Update codeblock delimiters per layout style guide

### ISSUE THREAD

**Issue by aconstas** [TheOdinProject/curriculum#26983]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

Codeblocks should be updated to align with new changes in our [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

In the lessons named in this issue, codeblocks will be opened by a set of 3 tildes (the ~ character) and then closed by another set of 3 tildes. To align with the style guide, these tilde characters need to change to backticks (the ` character). Be sure to refer to [the codeblocks section](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#codeblocks) of the style guide for an example of what this should look like.
## Guidance:
- Comment on this issue to get a lesson assigned to you. You can also ask for a specific one
- Please wait until you are assigned by a maintainer before starting to work on this issue to avoid multiple people working on the same lessons
- Make sure to read the [contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) before starting your first contribution
- Please add the following to your PR in the Issues section: Related to #xxxxx, since this issue is divided into sections, individual PRs should NOT INCLUDE the closes keyword.

### Authentication
- [x] @aconstas [Authentication Basics](https://www.theodinproject.com/lessons/nodejs-authentication-basics)

### Express and Mongoose
- [x] @VMadhuranga [Express 101](https://www.theodinproject.com/lessons/nodejs-express-101)
- [x] @Ahm3rIjaz [Project: Mini Message Board](https://www.theodinproject.com/lessons/nodejs-mini-message-board)

### Introduction to NodeJS
- [x] @jphamtv [Introduction: What is NodeJS?](https://www.theodinproject.com/lessons/nodejs-introduction-what-is-nodejs)

### Testing Express
- [x] @riarsdev [Testing Routes and Controllers](https://www.theodinproject.com/lessons/nodejs-testing-routes-and-controllers)
- [x] @MeDeveloperWeb [Testing Database Operations](https://www.theodinproject.com/lessons/nodejs-testing-database-operations)

@JoshDevHub

### Path

Node / JS

### Lesson Url

[nodeJs](https://www.theodinproject.com/paths/full-stack-javascript/courses/nodejs)

**Comment by VMadhuranga** [comment]
I would like to help

**Comment by Ahm3rIjaz** [comment]
I would like to work on this issue.

**Comment by JoshDevHub** [comment]
Thank you both for volunteering!

@VMadhuranga I've assigned you the Express 101 lesson

@Ahm3rIjaz I've assigned you the Mini Message Board project

**Comment by jphamtv** [comment]
Hi, I would like to help with this issue.

**Comment by JoshDevHub** [comment]
Thank you for volunteering @jphamtv 

I've assigned you the Introduction to Node JS lesson.

**Comment by riarsdev** [comment]
Count me in! I'm interested in contributing to resolve this.

**Comment by JoshDevHub** [comment]
Thank you for volunteering @riarsdev

I've assigned you the Testing Routs and Controllers lesson

**Comment by MeDeveloperWeb** [comment]
Hey! I would like to work on this issue.

**Comment by JoshDevHub** [comment]
Awesome @MeDeveloperWeb 

I've assigned you the Testing Database Operations method

**Comment by JoshDevHub** [comment]
Thanks everyone
